style/ruby/README.md

```diff
@@ -1,93 +1,4 @@
 Ruby
 ====
 
-[Sample](sample.rb)
-
-* Avoid conditional modifiers (lines that end with conditionals). [36491dbb9]
-* Avoid multiple assignments per line (`one, two = 1, 2`). [#109]
-* Avoid organizational comments (`# Validations`). [#63]
-* Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
-  instead to emphasize code branches. [36491dbb9]
-* Avoid explicit return statements. [#48]
-* Avoid using semicolons. [#108]
-* Avoid bang (!) method names. Prefer descriptive names. [#122]
-* Don't include spaces after `(`, `[` or before `]`, `)`.
-* Don't use `self` explicitly anywhere except class methods (`def self.method`)
-  and assignments (`self.attribute =`). [#89]
-* Don't vertically align tokens on consecutive lines.
-* If you break up a chain of method invocations, keep each method invocation on
-  its own line. Place the `.` at the end of each line, except the last.
-* If you break up a hash, keep the elements on their own lines and closing curly
-  brace on its own line.
-* Indent continued lines two spaces.
-* Indent private methods equal to public methods.
-* Name variables created by a factory after the factory (`user_factory`
-  creates `user`).
-* Prefer nested class and module definitions over the shorthand version
-  [Example][class definition example] [#332]
```

Comment:
Standard doesn't enforce this one, and I feel pretty strongly about keeping it: there are all kinds of weird constant lookup issues that we can run into when we don't do this.